examples/scale/backend/backend.go

```diff
@@ -39,17 +40,6 @@ var (
 	})
 
 	activeScenario = scenarios.ActiveScenario
-
```

Comment:
What's the reason for moving this into a separate package?  Just so the backfill scenario can access it?

Reply:
Yes, I moved metrics into separate package because I wanted to reuse them.